kevm-pyk/src/kontrol/solc_to_k.py

```diff
@@ -228,6 +237,7 @@ def application(self) -> KInner:
     raw_sourcemap: str | None
     methods: tuple[Method, ...]
     fields: FrozenDict
+    prefix_code: str = 'Z'
```

Comment:
If this is supposed to be a constant, I guess it should be marked accordingly. `PREFIX_CODE: Final = 'Z'`.

Reply:
Right, thanks!